src/components/NestedEditComponent.tsx

```diff
@@ -195,6 +195,25 @@ export class NestedEditComponent extends React.Component<InlineComponentProps, a
                     />
                 </div>
             )
+        } else if (field.type === "customedit") {
+            // const CustomEditComponent = field.customEditComponent({ currentModel: this.props.currentModel, additionalModels: this.props.additionalModels,
+            //     parentModel: this.props.parentModel, field })
+            const CustomEditComponent = field.customEditComponent
+            return (
+                <div>
+                    {!_.isEmpty(field.title) && <span><label style={{
+                        fontSize: "10px",
+                        marginRight: "10px"
+                    }}>{field.title.toUpperCase()}</label><br /></span>}
+                    <CustomEditComponent
+                        currentModel={this.props.currentModel}
+                        additionalModels={this.props.additionalModels}
+                        parentModel={this.props.parentModel}
+                        field={field}
+                        handleChange={this.select}
+                        model={this.props.currentModel}/>
+                </div>
+            )
```

Comment:
Added Support for Custom Editable Component. 
User can create his own component in his repo. It can linked via "customedit".